spec/spec_helper.cr

```diff
@@ -11,6 +11,9 @@ require "uri"
 
 FileUtils.rm_rf("/tmp/lavinmq-spec")
 
+FileUtils.rm_rf("./spec/resources/data_dir_index_v2_new")
+FileUtils.cp_r("./spec/resources/data_dir_index_v2", "./spec/resources/data_dir_index_v2_new")
```

Comment:
Have this as a `before_each` in the `context` block in Storage spec ?

Reply:
[d731eb4](https://github.com/cloudamqp/lavinmq/pull/423/commits/d731eb491e10f6fcf14aecd34caa32e2ba55b1dd) and [5a761e9](https://github.com/cloudamqp/lavinmq/pull/423/commits/5a761e9d28442de860168b7bc4d46bd9b759cc92)